atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/TableRemappingKeyValueService.java

```diff
@@ -460,4 +461,13 @@ public ClusterAvailabilityStatus getClusterAvailabilityStatus() {
     public boolean shouldTriggerCompactions() {
         return delegate.shouldTriggerCompactions();
     }
+
+    @Override
+    public ListenableFuture<Map<Cell, Value>> getAsync(TableReference tableRef, Map<Cell, Long> timestampByCell) {
+        try {
+            return delegate().getAsync(tableMapper.getMappedTableName(tableRef), timestampByCell);
+        } catch (TableMappingNotFoundException e) {
```

Comment:
This isn't necessarily correct, I feel like you need to also map the future's exception?

Reply:
There are no exceptions raised immediately during constructing the `getAsync` future. The previous doc for `getAsync` was wrong.